setup.py

```diff
@@ -0,0 +1,36 @@
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+with open(os.path.join(here, 'README.md')) as f:
```

Comment:
To prevent unicode decode errors in case we add Unicode characters in the README file, we can use this function: https://github.com/Kinto/kinto/blob/master/setup.py#L9-L13